### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by iwishiwasaneagle** [gTile/gTile#381] [General]
I just wanted to thank the developers for an amazing piece of software. I just transitioned from a dual monitor to a ultrawide setup and gtile has made this a breeze. Thanks!

**Comment by schnz** [comment]
Thank you! Much appreciated!

May I ask which version of the extension you are using? There was a breaking change in Gnome which required to develop a separate version of the extension. Now depending on the Gnome version you are either using the legacy version, or the newer one.

**Reply by iwishiwasaneagle** [reply]
Looks like I'm using version 59 which judging by the GH releases is the latest one
